src/scss/components/drawer/_index.scss

```diff
@@ -1,18 +1,19 @@
-@import (reference) 'vars';
+@import "vars";
 
```

Comment:
Please add `@import "../config";` to import where `%hxComponent` is defined.

Reply:
`@import` added